### PR TITLE
Realign with main repo

### PR DESCRIPTION
**Pull request by sjhood**

### CODE REVIEW

File: uptycs/models.go [hunk 1]
```diff
@@ -298,6 +298,28 @@ type Roles struct {
...
+	Name                          string        `json:"name,omitempty"`
+	Description                   string        `json:"description,omitempty"`
+	Custom                        bool          `json:"custom"`
+	Priority                      int           `json:"priority,omitempty"`
+	CreatedBy                     string        `json:"createdBy"`
+	UpdatedBy                     string        `json:"updatedBy"`
+	CreatedAt                     string        `json:"createdAt"`
+	UpdatedAt                     string        `json:"updatedAt"`
+	Links                         []LinkItem    `json:"links"`
+	ComplianceProfileObjectGroups []ObjectGroup `json:"complianceProfileObjectGroups"`
```
Comment (myoung34) [inline uptycs/models.go:312]: is this needed? I dont see it in the schema or swagger docs

File: uptycs/models.go [hunk 2]
```diff
@@ -298,6 +298,28 @@ type Roles struct {
...
+}
+
+type ComplianceProfiles struct {
+	Links      []LinkItem          `json:"links,omitempty"`
+	Items      []ComplianceProfile `json:"items,omitempty"`
+	Offset     int                 `json:"offset,omitempty"`
+	Limit      int                 `json:"limit,omitempty"`
+	Decorators []string            `json:"decorators,omitempty"`
```
Comment (myoung34) [inline uptycs/models.go:320]: Weird, this is unique to complianceProfiles and might be a bug. ask if this is actually real in the uptycs channel perhaps?

**Review by myoung34 — CHANGES_REQUESTED** [review]

File: uptycs/models.go (hunk 1 above)
Comment (sjhood) [reply uptycs/models.go:312]: Ah good catch, I was attempting to fix an error and put that in but it ended u being silly. I'll go ahead and remove it.

File: uptycs/models.go (hunk 2 above)
Comment (sjhood) [reply uptycs/models.go:320]: Messaged uptycs

**Review by myoung34 — APPROVED** [review]

**Pull request merged**